From d352df3380953ce7acb34aa26dc3b343853c32df Mon Sep 17 00:00:00 2001
From: Gunnar Beutner <gunnar@beutner.name>
Date: Mon, 19 Apr 2021 16:17:10 +0200
Subject: [PATCH 2/4] Use the correct include paths for SDL

The SDL port is installed into /usr/local, and its headers are
accessible as <SDL2/...>.
---
 src/types.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/types.h b/src/types.h
index 1f696d1..b5ede07 100644
--- a/src/types.h
+++ b/src/types.h
@@ -31,8 +31,8 @@ The authors of this program may be contacted at https://forum.princed.org
 // These headers for SDL seem to be the pkgconfig/meson standard as per the
 // latest versions. If the old ones should be used, the ifdef must be used
 // to compare versions. 
-# include <SDL.h>
-# include <SDL_image.h>
+# include <SDL2/SDL.h>
+# include <SDL2/SDL_image.h>
 //#endif
 
 #if SDL_BYTEORDER != SDL_LIL_ENDIAN
-- 
2.36.1

